### PR TITLE
Added DG under resources and forum section

### LINKED ISSUE #204: Forum component class diagram

### PR DESCRIPTION
**Pull request by Rrraaaeee**
Fixes #204

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -183,15 +182,15 @@ Apart from `Cooper`, the rest of the app consists of these seven components:
...
+> ℹ️`userInput` represents the credentials input by the user for verification. For example, `register John /pw 12345 /as admin`.
 
 <p align="center">
     <img src="developerGuideDiagrams/signInSequenceDiagram.png" alt="signInSequenceDiagram"><br>
 </p> 
 
 - The next sequence diagram below shows how cOOPer's components interact with each other when a user enters a **command** at the _features layer_.
 
-> ℹ️ `userInput` represents a command input by the user. For example, `meetings`.<br>
+> ℹ️ `userInput` represents a command input by the user. For example, `meetings`.
```
Comment (theeugenechong) [inline docs/DeveloperGuide.md:193]: need the line break here or there will be formatting issues

Comment (Rrraaaeee) [reply docs/DeveloperGuide.md:193]: Weird it looks alright on typora. I will add them back then

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -97,7 +97,6 @@ This section includes the sources of code, documentation and third-party librari
    1. Navigate to `src/main/java/cooper/Cooper.java`
    2. Right click on `Cooper.java` and select 'Run Cooper.main()'.
    3. You should see the following output if the setup was done correctly:
-
```
Comment (theeugenechong) [inline docs/DeveloperGuide.md:100]: Oh ya and also I think we need the line here also, I remember my ip UG nearly got screwed up cause i didn't add the extra empty line here

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (theeugenechong) [reply docs/DeveloperGuide.md:193]: Hey I'll merge it first, and see how the website looks like

**Pull request merged**